webpack.config.js

```diff
@@ -12,18 +12,27 @@ const pathToOwnWebpackConfig = join(context, "webpack.config.js");
 const hasOwnWebpackConfig = existsSync(pathToOwnWebpackConfig);
 const webpackConfigToMerge = hasOwnWebpackConfig ? require(pathToOwnWebpackConfig) : {};
 
-const package = JSON.parse(readFileSync(join(context, "package.json")));
+/**
+ * Return the current version of the app.
+ *
+ * Format: `${package.json:version}-${gitShortSha}`
+ * or just the package version if git is not available.
+ */
+function getVersion() {
+  const package = JSON.parse(readFileSync(join(context, "package.json")));
+  let version = package.version;
 
-let gitVersion = package.version;
+  try {
```

Comment:
How about 

```js
try {
  return `${package.version}-${execSync("git rev-parse --short HEAD").toString().trim()}`
} catch (e) {
  console.log(e.message);
}
```

I don't really see why this implementation needs to be more verbose than this.

Reply:
I can also do:
```js
try {
  return `${package.version}-${execSync("git rev-parse --short HEAD").toString().trim()}`
} catch (e) {
  console.log(e.message);
  return package.version
}
```

If you prefer :wink:

Reply:
Yes. I prefer that because it is easier to follow the intent behind it and does not involve storing bits of mutable state around the implementation.